### PR TITLE
Make `CommentMarkdownTextFlowContainer` render images

### LINKED ISSUE #19688: Images not showing in lazer client
### Type

Game behaviour

### Bug description

Lazer doesnt show images on the Beatmap download page when clicking on the preferred map and scrolling to the comments while somebody posted an image
osu!lazer

### Screenshots or videos

osu!lazer
image

Website
image

### Version

2022.810.2-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9301061/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9301062/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9301063/network.log)
[performance.log](https://github.com/ppy/osu/files/9301064/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9301065/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9301066/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9301067/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9301068/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9301069/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9301072/updater.log)

### PR DESCRIPTION
**Pull request by bassepeder**
Closes https://github.com/ppy/osu/issues/19688.

### CODE REVIEW

File: osu.Game/Overlays/Comments/CommentMarkdownContainer.cs [hunk 1]
```diff
@@ -18,8 +18,10 @@ public class CommentMarkdownContainer : OsuMarkdownContainer
 
         private class CommentMarkdownTextFlowContainer : OsuMarkdownTextFlowContainer
         {
-            // Don't render image in comment for now
-            protected override void AddImage(LinkInline linkInline) { }
+            protected override void AddImage(LinkInline linkInline)
+            {
+                AddDrawable(new OsuMarkdownImage(linkInline));
+            }
         }
```
Comment (peppy) [inline osu.Game/Overlays/Comments/CommentMarkdownContainer.cs:25]: I'm pretty sure all you need to do is remove this class and the place it's made completely. What you've done here is reimplemented this method in exactly the same way that it's implemented in the base class.

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Game/Overlays/Comments/CommentMarkdownContainer.cs (hunk 1 above)
Comment (bastianpedersen) [reply osu.Game/Overlays/Comments/CommentMarkdownContainer.cs]: I've removed the `CommentMarkdownTextFlowContainer` class and rather use the base class `OsuMarkdownTextFlowContainer`.

Comment (peppy) [reply osu.Game/Overlays/Comments/CommentMarkdownContainer.cs]: Please don't mark things resolved, leave that to us.

File: osu.Game/Overlays/Comments/CommentMarkdownContainer.cs [hunk 2]
```diff
@@ -4,24 +4,17 @@
...
-using Markdig.Syntax.Inlines;
 using osu.Framework.Graphics.Containers.Markdown;
 using osu.Game.Graphics.Containers.Markdown;
 
 namespace osu.Game.Overlays.Comments
 {
     public class CommentMarkdownContainer : OsuMarkdownContainer
     {
-        public override MarkdownTextFlowContainer CreateTextFlow() => new CommentMarkdownTextFlowContainer();
+        public override MarkdownTextFlowContainer CreateTextFlow() => new OsuMarkdownTextFlowContainer();
```
Comment (peppy) [inline osu.Game/Overlays/Comments/CommentMarkdownContainer.cs:14]: This line is not required. Please take a moment to observe the object inerhitence.

**Review by peppy — APPROVED** [review]

**Pull request merged**